pkg/cloud/services/ec2/instances.go

```diff
@@ -1119,6 +1122,22 @@ func filterGroups(list []string, strToFilter string) (newList []string) {
 	return
 }
 
+func getCapacityReservationSpecification(capacityReservationID string) *ec2.CapacityReservationSpecification {
+	if capacityReservationID == "" {
+		// Instance is not a CapacityReservation instance
```

Comment:
See above. There's "[EC2] instance" and "Capacity Reservation" as two separate terms. Let's not mix those.
```suggestion
		// Not targetting any specific Capacity Reservation
```

Reply:
ok